contentctl/actions/detection_testing/views/DetectionTestingView.py

```diff
@@ -93,16 +94,27 @@ def getSummaryObject(
             )
 
             # Aggregate detection pass/fail metrics
-            if summary["success"] is True:
-                total_pass += 1
-            else:
+            if summary["success"] is False:
                 total_fail += 1
+            else:
+                #Test is marked as a success, but we need to determine if there were skipped unit tests
+                #SKIPPED tests still show a success in this field, but we want to count them differently
+                pass_increment = 1
+                for test in summary.get("tests"):
+                    if test.get("test_type") == "unit" and test.get("status") == "skip":
+                        total_skipped += 1
+                        #Test should not count as a pass, so do not increment the count
+                        pass_increment = 0
+                        break
+                total_pass += pass_increment
+                
 
             # Append to our list
             tested_detections.append(summary)
 
         # Sort s.t. all failures appear first (then by name)
-        tested_detections.sort(key=lambda x: (x["success"], x["name"]))
+        #Second short condition is a hack to get detections with unit skipped tests to appear above pass tests
```

Comment:
Can I ask why you want skip above pass?

Reply:
The results file can be, and often is, REALLY large.  I think it makes sense for errors/failures at the top, as these are the ones that deserve the most attention (and what people are likely to be looking for when they open this file).

I _personally_ think "pass" is the _least_ interesting since it means they are working as expected.
I think SKIP falls somewhere in between a failure and a pass.  For example, someone might want to manually evaluable these OR note how may tests have been skipped.

Definitely a personal opinion.